subprojects/gradle-guides-plugin/src/main/java/org/gradle/docs/samples/Sample.java

```diff
@@ -16,10 +17,17 @@
      */
     DirectoryProperty getSampleDirectory();
 
+    /**
+     * By Convention, this is README.adoc within the sample directory.
+     *
+     * @return Property for configuring the readme file for the sample in Asciidoctor format.
+     */
+    RegularFileProperty getReadme();
```

Comment:
To make this consistent with other properties, can you make this `getReadmeFile`?

Reply:
Fixed.